gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/integration/use_case/IngestIntegrationApisUseCase.java

```diff
@@ -66,6 +67,15 @@ public class IngestIntegrationApisUseCase {
     private final CreateApiDocumentationDomainService createApiDocumentationDomainService;
     private final LicenseDomainService licenseDomainService;
 
+    private static final EnumMap<IntegrationApi.Validation, PlanValidationType> VALIDATION_MAPPING = new EnumMap<>(
+        IntegrationApi.Validation.class
+    );
+
```

Comment:
Shouldn't we make this mapping part of IntegrationAdapter?

Reply:
We can't because it's break the onion architecture